README.md

```diff
@@ -124,7 +124,7 @@ Jenkins.getInstance().clouds.add(myCloud)
 [Azure Container Instances](https://docs.microsoft.com/en-us/azure/container-instances/) offers the fastest and simplest way to run a container in Azure, without having to provision any virtual machines and without having to adopt a higher-level service.
 
 ## Pre-requirements
-* Resource Group in West US, East US or West Europe. Get [region availability details](https://azure.microsoft.com/en-us/regions/services/).
+* Resource Group in available regions. Get [region availability details](https://azure.microsoft.com/en-us/regions/services/).

```

Comment:
available for ACI? so the link should be with query: `https://azure.microsoft.com/en-us/global-infrastructure/services/?products=container-instances`

Reply:
done